tests/webpack_stats_test.ts

```diff
@@ -0,0 +1,18 @@
+import fs = require('fs');
+import path = require('path');
+
+import webpack_stats = require('../src/webpack_stats');
+const { isMultiCompilation } = webpack_stats;
+
+describe('isMultiCompilation()', () => {
+	it('should work for non-multi stats objects', () => {
```

Comment:
I think "should return false" and "should return true" would be slightly clearer descriptions than "should work"